packages/cosmic-swingset/lib/ag-solo/vats/vat-priceAuthority.js

```diff
@@ -1,18 +1,12 @@
 import { Far } from '@agoric/marshal';
 import { makePriceAuthorityRegistry } from '@agoric/zoe/tools/priceAuthorityRegistry';
 import { makeFakePriceAuthority } from '@agoric/zoe/tools/fakePriceAuthority';
-import { makeLocalAmountMath } from '@agoric/ertp';
 
 export function buildRootObject(_vatPowers) {
   return Far('root', {
     makePriceAuthority: makePriceAuthorityRegistry,
     async makeFakePriceAuthority(options) {
-      const { issuerIn, issuerOut } = options;
-      const [mathIn, mathOut] = await Promise.all([
-        makeLocalAmountMath(issuerIn),
-        makeLocalAmountMath(issuerOut),
-      ]);
-      return makeFakePriceAuthority({ ...options, mathIn, mathOut });
+      return makeFakePriceAuthority({ ...options });
```

Comment:
```suggestion
      return makeFakePriceAuthority(options);
```
unless you're copying it for a reason. When a piece of code has no obvious purpose, at least a comment is nice.